includes/sanitizers/class-amp-tag-and-attribute-sanitizer.php

```diff
@@ -1022,6 +1002,12 @@ private function delegated_sanitize_disallowed_attribute_values_in_node( $node,
 				continue;
 			}
 
+			// Check the context to see if we are currently within a template tag.
+			// If this is the case and the attribute value contains a template placeholder, we skip sanitization.
+			if ( ! empty( $this->open_elements['template'] ) && preg_match( '/^.*{{.*}}/', $attr_node->nodeValue ) ) {
```

Comment:
The logic is not correct here. The `preg_match()` would only be correct for `type=amp-mustache`, any other type of template would probably have other placeholder syntax.